### ISSUE TITLE
https not working

### ISSUE THREAD

**Issue by priema** [ichiTechs/Dockerized-SSL-NextCloud-with-MariaDB#1]
I followed the readme but after executing these commands I was not able to access nextcloud with https, only http worked.
the content of my docker:

CONTAINER ID        IMAGE                                    COMMAND                  CREATED             STATUS              PORTS                                      NAMES
43be44891cb2        nginx                                    "nginx -g 'daemon of…"   12 minutes ago      Up 12 minutes       80/tcp                                     nextcloud_webserver
450f0312ada7        jrcs/letsencrypt-nginx-proxy-companion   "/bin/bash /app/entr…"   12 minutes ago      Up 12 minutes                                                  letsencrypt-companion
b53204c88934        nextcloud:fpm                            "/entrypoint.sh php-…"   12 minutes ago      Up 12 minutes       9000/tcp                                   nextcloud_fpm
4a91e2151140        jwilder/nginx-proxy                      "/app/docker-entrypo…"   12 minutes ago      Up 12 minutes       0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp   proxy
e0fba24177e7        mariadb                                  "docker-entrypoint.s…"   12 minutes ago      Up 12 minutes       3306/tcp                                   db

you can see that nginx is only running on port 80. how can I fix that?

**Comment by dianjuar** [comment]
Hello. 
I implemented the letsencrypt services to do it with a WordPress environment following this approach and everything perfect, the repo is https://github.com/bikecoders/docker-compose_wordpress, you check it to have reference.

Remember that it only works on productions environments. Not localhost.

**Comment by priema** [comment]
I tested it with my domain and it is not working. The port 443 is not open on the nginx docker only on the proxy.

the ip 192.168.1.7 is my server running the docker

➜  / nc -zv 192.168.1.7 443
nc: connectx to 192.168.1.7 port 443 (tcp) failed: Connection refused

➜  / nc -zv 192.168.1.7 80  
found 0 associations
found 1 connections:
     1:	flags=82<CONNECTED,PREFERRED>
	outif en0
	src 192.168.1.8 port 49264
	dst 192.168.1.7 port 80
	rank info not available
	TCP aux info available

Connection to 192.168.1.7 port 80 [tcp/http] succeeded!

what I also noticed is that the docker does not automatically create a proxy-tier network. it creates a network called "dockerizedsslnextcloudwithmariadb_default" as bridge local..
